app.rb

```diff
@@ -0,0 +1,57 @@
+require 'sinatra'
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/message'
+require './lib/user'
+
+class ChitterController < Sinatra::Base
+
+  enable :sessions
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/flow' do
+    # @user = User.find(session[:user_id])
```

Comment:
Is the commented out code needed in the future? If not, it would be best to remove.

Reply:
Thanks for pointing this out - hashed out code needs to be deleted.